Dockerfile

```diff
@@ -16,8 +19,10 @@ FROM library/alpine:3.7
 RUN apk add --no-cache --update ca-certificates
 
 COPY --from=build /out/caddy /usr/local/bin/caddy
+RUN mkdir -p /var/lib/caddy \
+  && touch /var/lib/caddy/Caddyfile
 
 EXPOSE 80 443 2015
 ENTRYPOINT ["/usr/local/bin/caddy"]
 CMD ["-agree", "-conf", "/var/lib/caddy/Caddyfile"]
-
+#, "--type", "net"]
```

Comment:
Did this mean to be left in? If it's an optional, maybe worth making this an optional build argument to use?